src/include/daos/common.h

```diff
@@ -836,7 +836,11 @@ struct daos_hhash_table {
 extern struct daos_hhash_table daos_ht;
 
 /* daos handle hash table helpers */
-int daos_hhash_init(void);
+int daos_hhash_init_feats(uint32_t feats);
+static inline int daos_hhash_init()
```

Comment:
(style)  Bad function definition - int daos_hhash_init() should probably be int daos_hhash_init(void)